mongodb/src/main/java/org/apache/calcite/adapter/mongodb/MongoProject.java

```diff
@@ -57,8 +58,7 @@ public MongoProject(RelOptCluster cluster, RelTraitSet traitSet,
 
   @Override public Project copy(RelTraitSet traitSet, RelNode input,
       List<RexNode> projects, RelDataType rowType) {
-    return new MongoProject(getCluster(), traitSet, input, projects,
-        rowType);
+    return new MongoProject(getCluster(), traitSet, input, projects, rowType);
```

Comment:
don't we need any specific test cases for modules other than core module that are impacted?